File: NodeStarter/UserStories/US01.js
const get_results = (left, right, sum) => {
    if(left + right != sum)
        return [left, right, sum];
    return [];
}

const print_results = (left, right, sum) => {
    console.log(`ERROR US01: ${left} and ${right} do not sum to ${sum}`);
}

module.exports = {
    get_results,
    print_results
}

File: RubyStarter/UserStories/US01.rb
class US01

    def get_results(left, right, sum)
        if(left + right != sum)
            return [left, right, sum]
        end
        return []
    end

    def print_results(lst)
        puts "ERROR US01: #{lst[0]} and #{lst[1]} do not sum to #{lst[2]}"
    end

end